### DISCUSSION TITLE
Namespaced keys are not read from database

### DISCUSSION THREAD

**Discussion by bilogic** [spatie/laravel-translation-loader#168] [Bugs]
https://github.com/spatie/laravel-translation-loader/blob/b45980636d3d60a3426ac44f5a51517bca8cb63a/src/TranslationLoaderManager.php#L25-L28

https://github.com/spatie/laravel-translation-loader/blob/b45980636d3d60a3426ac44f5a51517bca8cb63a/README.md?plain=1#L13-L15

> You can even mix using language files and the database. If a translation is present in both a file and the database, the database version will be returned.

Unlike as described in the `README.md`, why are namespaced keys read from files only and not from the DB?

**Comment by aon4o** [comment]
I have tried to implement the `namespaced translations` feature here - #174 .

**Reply by bilogic** [reply]
thanks
